### PR TITLE
Fix for issue#120

### PR DESCRIPTION
**Pull request by maxsocl**
@cjlee112 @VladimirFilonov 

This PR adds fix for issue #120.

Tests will be added only after PR #60 will be merged.

### CODE REVIEW

File: mysite/ct/views.py [hunk 1]
```diff
@@ -898,6 +900,7 @@ def unit_resources(request, course_id, unit_id):
 @login_required
 def wikipedia_concept(request, course_id, unit_id, source_id):
     'page for viewing or adding Wikipedia concept to this courselet'
```
Comment (sengottuvelk) [inline mysite/ct/views.py]: We can adopt the standard `"""docstring"""` format here.

**Comment by sengottuvelk** [comment]
LGTM. Merge when ready.

File: mysite/ct/views.py (hunk 1 above)
Comment (maxsocl) [inline mysite/ct/views.py]: > We can adopt the standard """docstring""" format here.

It is an old code and not related to this PR.
Sure, we should refactor this but not in this PR - different PR type should be separated (feature, refactor old code etc.)

Comment (sengottuvelk) [inline mysite/ct/views.py]: ok

**Pull request merged**